other/upgrade_2-1_mysql.sql

```diff
@@ -2441,3 +2441,10 @@ if ($doChange)
 ---# Add the new log banned ip
 ALTER TABLE {$db_prefix}log_online ADD COLUMN ip VARBINARY(16);
 ---#
+
+--# Add start_time and end_time support to the calendar
```

Comment:
you need three dashes, not two

https://github.com/SimpleMachines/SMF2.1/wiki/SQL-file-for-upgrade